Add .travis.yml for Continuous Integration

theparadoxer02 opened the pull request:
### Add Continuous Intgration Feature

#### Current features include :-
  - Check s Style Guide for whole project using Flake8 and in case of errors, display them all
  - Run tests using pytest
  - Checks if `makemigrations` command work
#

sourabhtk37 commented:
Few things:
- Please add description for your changes.
- Remove the existing `.travis.yml` file in the tree

theparadoxer02 committed:
Add Continuous Intgration Feature.

Current features include:
 - Check Style Guide for whole project using Flake8 and in case of errors, display them all
 - Run tests using pytest
 - Check if `makemigrations` command work

theparadoxer02 commented:
@sourabhtk37 i have made some changes, please have a look at it now.
Also if anything needed to add in this feature then please suggest :)

sourabhtk37 reviewed (approved):
Lgtm
